autopilot/networking-tutorial/dns.tf

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform {
+  required_version = "~> 1.3"
+}
+
+provider "google" {}
+
+variable "base_domin" {
+  type        = string
+  description = "Your base domain"
+}
+
+variable "name" {
+  type        = string
+  description = "Name of resources"
+  default     = "networking-tutorial"
+}
+
+data "google_client_config" "current" {}
+
+resource "google_compute_global_address" "default" {
+  name = var.name
```

Comment:
Does this mean that the global IP address that gets created will have the standard name "networking-tutorial"? If so I should change this to match the IP address referenced in kubernetes-manifests.yaml.